src/test/resources/com/teamscale/upload/output.xcresult.tar.gz.xccov.expected

```diff
@@ -1,4 +1,4 @@
-/Users/albert/repos/test/xcresultcovnersion/XCResultConversion/Tests/ObjectiveCTest.m
+/Users/albert/repos/test/xcresultcovnersion/XCResultConversion/Tests/ObjectiveCTest.m:
```

Comment:
For the reviewer: Trailing colons together with the newlines are the output format of the conversion since XCode 13.3. Tests seem to have used old format before